Atomic/atomic.py

```diff
@@ -731,14 +734,24 @@ def get_col_lengths(_images):
                 if image['Id'] in vuln_ids:
                     space = " " if len(indicator) < 1 else ""
                     indicator = indicator + self.skull + space
+                    image_dict["vulnerable"] = True
+                else:
+                    image_dict["vulnerable"] = False
 
                 image_id = image["Id"][:12] if self.args.truncate else image["Id"]
                 image_type = image['ImageType']
-                util.write_out(col_out.format(indicator, repo,
-                                              tag, image_id,
-                                              created,
-                                              virtual_size,
-                                              image_type))
+                image_dict["repo"] = repo
+                image_dict["tag"] = tag
+                image_dict["id"] = image["Id"]
+                image_dict["created"] = created
+                image_dict["virtualsize"] = virtual_size
+                image_dict["type"] = image_type
+                image_dict["vuln_info"] = json.loads(all_vuln_info)[image["Id"]]
+                all_image_info.append(image_dict)
+                if self.useTTY:
+                    util.write_out(col_out.format(indicator, repo, tag, image_id, created, virtual_size, image_type))
+            if not self.useTTY:
+                return json.dumps(all_image_info)
             util.write_out("")
```

Comment:
so have the sing_all_image_info function call the images function which returns the json string which is then properly parsed so we can get the information and then display it?